scripts/worktree/wt-common.sh

```
@@ -0,0 +1,120 @@
+  "files.eol": "\\n",
+  "files.insertFinalNewline": true,
+  "files.trimFinalNewlines": true,
+  "files.trimTrailingWhitespace": true,
+  "[markdown]": { "files.trimTrailingWhitespace": false },
+
+  // --- TypeScript ---
+  "typescript.preferences.importModuleSpecifier": "non-relative",
+
+  // --- Tailwind IntelliSense (if using the extension) ---
+  "tailwindCSS.experimental.classRegex": [
+    "['\"`]([^'\"`]*tw-[^'\"`]*)['\"`]",
```

Shellcheck (0.11.0)

[warning] Found trailing ] outside test. Add missing [ or quote if intentional.

(SC2171)